### PR TITLE
fix for depstat's periodical prow job

### PR DESCRIPTION
**Pull request by RinkiyaKeDad**
Signed-off-by: RinkiyaKeDad <arshsharma461@gmail.com>

I modified the original script to just run install depstat and run it with the `--json` flag (instead of redirecting the output to a file like earlier). Is this what you had in mind @dims?

/assign @dims

### CODE REVIEW

File: config/jobs/kubernetes/sig-arch/kubernetes-depstat-periodical.yaml [hunk 1]
```diff
@@ -21,11 +21,8 @@ periodics:
           pushd "$ARTIFACTS"; \
           go install github.com/kubernetes-sigs/depstat@latest; \
           popd; \
-          depstat stats --json > "${ARTIFACTS}/stats.json"; \
```
Comment (dims) [inline config/jobs/kubernetes/sig-arch/kubernetes-depstat-periodical.yaml:24]: we should print the output and store it into a file as well. will be useful for later automation, look at the `tee` command in linux

File: config/jobs/kubernetes/sig-arch/kubernetes-depstat-periodical.yaml [hunk 2]
```diff
@@ -21,7 +21,7 @@ periodics:
           pushd "$ARTIFACTS"; \
           go install github.com/kubernetes-sigs/depstat@latest; \
           popd; \
-          depstat stats --json || true;
```
Comment (dims) [inline config/jobs/kubernetes/sig-arch/kubernetes-depstat-periodical.yaml]: do you really need `|| true`? is the depstat command expected to fail, i'd rather see RED when it does.

**Comment by RinkiyaKeDad** [comment]
Forgot to ping you on this one @dims 😅 can we merge this now?

**Pull request merged**